decorator.rb

```diff
@@ -0,0 +1,28 @@
+require_relative './nameable'
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.capitalize
+  end
+end
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    word_output = @nameable.correct_name
+    if word_output.length <= 10
+      word_output
+    else
+      word_output[0..10]
+    end
+  end
```

Comment:
- [ ] **[OPTIONAL]** I think it would be better to put these classes in different files.

Reply:
@julie-ify please can you specify to the next reviewer that this is optional. Thank you.